tests/FakeItEasy.Analyzer.Tests/Helpers/DiagnosticVerifier.cs

```diff
@@ -114,9 +81,9 @@ private static void VerifyDiagnosticResults(IEnumerable<Diagnostic> actualResult
                     {
                         var message =
                             FormattableString.Invariant(
-                                $@"Expected:
-A project diagnostic with No location
-Actual:
+                                $@"Expected:
+A project diagnostic with No location
+Actual:

```

Comment:
@blairconrad I assume @thomaslevesque isn't using it, otherwise it wouldn't have let him save the LF's?